dev-tools/omdb/src/bin/omdb/sled_agent.rs

```diff
@@ -72,22 +72,6 @@ enum BootstoreCommands {
     Status,
 }
 
-#[derive(Debug, Args)]
```

Comment:
Seems unrelated to this PR but whatever

Reply:
Ahh yeah, I was searching for variants of "chicken" and this _file_ came up, and I realized these should've been removed a while ago.